packages/language-server/src/server.ts

```diff
@@ -67,6 +77,11 @@ connection.onInitialize((params: InitializeParams) => {
         triggerCharacters: ['"', "'", ":", "<"],
       },
       hoverProvider: true,
+      codeActionProvider: true,
+      // Each command executes a different code action scenario
+      executeCommandProvider: {
+        commands: [QUICK_FIX_STABLE_ID],
```

Comment:
Do we need to ensure uniqueness of command names?
If our name is not very unique, could the command ever be executed due to another cause (e.g enother plugin)?

How do other extensions provide commands? is there a unique prefix, e.g: `ui5_language_assistant.quick_fix_stable_id` ?